src/PeriodEnd/CoInvestedPayments/DeployCoInvestedPayments/sql/dml/03 PeriodEnd.Populate.Reference.PaymentsHistory.dml.sql

```diff
@@ -22,5 +22,11 @@ SELECT
 	JOIN  ${DAS_PeriodEnd.FQ}.PaymentsDue.RequiredPayments rp on p.RequiredPaymentId = rp.Id
     WHERE rp.Ukprn IN (SELECT DISTINCT [Ukprn] FROM [Reference].[Providers])
 	AND p.FundingSource != 1
+	AND p.[CollectionPeriodName] IN 
```

Comment:
We could add refactor the query to make it "slightly" more performant. However, as we are joining on a remote database (DEDS) that we have no control over the schema for, there isn't a massive benefit doing much to our local join queries at this stage.